### PR TITLE
User show page--Initial layout and setup

### LINKED ISSUE #55: User Story 17, User Profile Show Page
```
As a registered user
When I visit my own profile page
Then I see all of my profile data on the page except my password
And I see a link to edit my profile data
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #56

### PR DESCRIPTION
**Pull request by saleratus**
Added test for user seeing their info on the page, and a link to edit profile info
All tests passing.
Closes #55

### CODE REVIEW

File: spec/features/user_show_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+      visit user_path(user_1)
+
+      expect(page).to have_content(user_1.name)
+      expect(page).to have_content(user_1.street)
+      expect(page).to have_content(user_1.city)
+      expect(page).to have_content(user_1.state)
+      expect(page).to have_content(user_1.zip)
+      expect(page).to have_content(user_1.email)
+
```
Comment (danbriechle) [inline spec/features/user_show_page_spec.rb]: lets add a test to not expect password.
